superset-frontend/src/components/ImportModal/index.tsx

```diff
@@ -20,32 +20,34 @@ import React, { FunctionComponent, useEffect, useRef, useState } from 'react';
 import { styled, t } from '@superset-ui/core';
 
 import Icon from 'src//components/Icon';
-import Modal from 'src/common/components/Modal';
+import StyledModal from 'src/common/components/Modal';
 import { useImportResource } from 'src/views/CRUD/hooks';
 import { ImportResourceName } from 'src/views/CRUD/types';
 
 export const StyledIcon = styled(Icon)`
   margin: auto ${({ theme }) => theme.gridUnit * 2}px auto 0;
 `;
 
+const HelperMessage = styled.div`
+  display: block;
```

Comment:
curious, is there something that is inherited that is making this div some other kind of display, since `block` is the default?

Reply:
Ah, you're right, it's not needed here. I copied the style from another helper class where the display is `flex`.